.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Checklist
+
+- [ ] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
+- [ ] The description lists all applicable issues this PR seeks to resolve
+- [ ] The description lists any configuration setting(s) that need to be enabled/disabled in order for the software to run properly with these changes
```

Comment:
Maybe we should clarify this point to say "any configuration setting(s) that differ from the default settings."